packages/extensions/openapi-to-typespec/test/utils/generate-typespec.ts

```diff
@@ -24,6 +37,18 @@ export async function generateTypespec(repoRoot: string, folder: string, debug =
   generate(repoRoot, swaggerPath, debug, isFullCompatible);
 }
 
+export async function generateSwagger(folder: string) {
+  if (brownFieldProjects.includes(folder)) {
+    console.log(`Skipping brown field project ${folder}`);
+    return;
+  }
+
+  const { path: root } = await resolveProject(__dirname);
+  const path = join(root, "test", folder, "tsp-output");
+  const command = "tsp compile . --emit=@azure-tools/typespec-autorest";
+  execSync(command, { cwd: path, stdio: "inherit" });
```

Comment:
once we generated the swagger in the test project folder, will we keep them in the test project folder as well?

Reply:
Yes!